Cesium.CodeGen/Contexts/AssemblyContext.cs

```diff
@@ -165,12 +165,38 @@ public FieldReference GetConstantPoolReference(string stringConstant)
         Debug.Assert(writtenBytes == bufferSize - 1);
 
         var type = GetStubType(bufferSize);
-        field = GenerateFieldForStringConstant(type, data);
-        _stringConstantHolders.Add(stringConstant, field);
+        field = GenerateFieldForDataConstant(type, data);
+        _dataConstantHolders.Add(stringConstant, field);
 
         return field;
     }
 
+    public FieldReference GetConstantPoolReference(byte[] dataConstant)
+    {
+        if (dataConstant.Length >= 10)
+        {
+            // Do not attempt to put large arrays in cache.
+            // Most likely they are single use in the application.
+            // No chance that they would be cached by values.
+            var bufferSize = dataConstant.Length;
+            var type = GetStubType(bufferSize);
+            return GenerateFieldForDataConstant(type, dataConstant);
+        }
```

Comment:
To be honest, I can't see what do we lose if we avoid putting these into the `_dataConstantHolders`. Barely anything, really. We may spend a little more time on compilation (though if we hash the values in `O(N)` and cache the resulting hash values, then there's zero lose, algorithmically), while we may save little by reusing the same data, if it happens.

I'd say let's avoid this optimization for now.